dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/Body.java

```diff
@@ -0,0 +1,18 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import java.util.List;
+
+/**
+ * It's a {@link com.dotmarketing.portlets.templates.model.Template}'s Body
+ */
+public class Body {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)